docs/book/src/multiversion-tutorial/testdata/project/Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY api/ api/
 COPY controllers/ controllers/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

Comment:
go 1.16 seems to have `GO111MODULE=on` by default. But I think it should be better to explicitly have this.